### PR TITLE
Feature/python get functions/steffi

### PR DESCRIPTION
**Pull request by steffitan23**
Add get functions to CLI in get.py

### CODE REVIEW

File: api/api.go [hunk 1]
```diff
@@ -62,6 +63,370 @@ func (serv *MetadataServer) CreateUser(ctx context.Context, user *pb.User) (*pb.
 	return serv.meta.CreateUser(ctx, user)
 }
 
+// rpc GetUsers(stream Name) returns (stream User);
```
Comment (sdreyer) [inline api/api.go:66]: What are these doing?

File: client/src/featureform/cli.py [hunk 2]
```diff
@@ -37,24 +39,152 @@ def cli():
...
+        credentials = grpc.ssl_channel_credentials()
+        channel = grpc.secure_channel(host, credentials)
+    stub = ff_grpc.ApiStub(channel)
+
+    match resource_type:
+        case "users":
+            pass
+        case "features":
+            pass
+            # print()
```
Comment (sdreyer) [inline client/src/featureform/cli.py:168]: remove

File: client/src/featureform/cli.py [hunk 3]
```diff
@@ -37,24 +39,152 @@ def cli():
...
+              help="The host address of the API server to connect to")
+@click.option("--cert",
+              "cert",
+              required=False,
+              help="Path to self-signed TLS certificate")
+@click.option("--insecure",
+              is_flag=True,
+              help="Disables TLS verification")
+@click.argument("resource_type", required=True)
+def list(host, cert, insecure, resource_type):
```
Comment (sdreyer) [inline client/src/featureform/cli.py:139]: Comment this out for now if its not ready

File: api/api.go (hunk 1 above)
Comment (steffitan23) [reply api/api.go:66]: removed

**Pull request merged**